app/controllers/v0/disability_compensation_forms_controller.rb

```diff
@@ -19,7 +19,7 @@ def rated_disabilities
       application = params['application'] || 'form526'
       settings = Settings.lighthouse.veteran_verification[application]
       service = ApiProviderFactory.rated_disabilities_service_provider(
-        @current_user
+        { icn: @current_user.icn.to_s, auth_headers: }
```

Comment:
should the second key of `auth_headers` be pointing to something? Or is this syntactic sugar that I'm not aware of?